spyt-package/src/main/python/spyt/conf.py

```diff
@@ -26,6 +18,71 @@
 logger = logging.getLogger(__name__)
 
 
+class SpytDistributions:
+    def __init__(self, client: YtClient, yt_root: str):
+        self.client = client
+        self.yt_root = YPath(yt_root)
+        self.conf_base_path = self.yt_root.join("conf")
+        self.global_conf = client.get(self.conf_base_path.join("global"))
+        self.distrib_base_path = self.yt_root.join("distrib")
+        self.spyt_base_path = self.yt_root.join("spyt")
+
+    def read_remote_conf(self, cluster_version):
+        version_conf = self.client.get(self._get_version_conf_path(cluster_version))
+        version_conf["cluster_version"] = cluster_version
+        return update_inplace(self.global_conf, version_conf)  # TODO(alex-shishkin): Might cause undefined behaviour
+
+    def latest_ytserver_proxy_path(self, cluster_version):
+        if cluster_version:
+            return None
+        global_conf = self.global_conf
```

Comment:
This local variable is redundant, you can directly use `self.global_conf` on the next line